src/main/java/training/commands/ArmCommands/wristCommands/WristDefaultCommand.java

```diff
@@ -0,0 +1,20 @@
+package training.commands.ArmCommands.wristCommands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import training.subsystems.ArmSubsystems.wristSubsystem.Wrist;
+
+public class WristDefaultCommand extends Command {
```

Comment:
```suggestion
public class StopWrist extends Command {
```
same as elbow default command